PowerShellSyntax.tmLanguage

```diff
@@ -248,7 +248,13 @@
 		</dict>
 		<dict>
 			<key>match</key>
-			<string>(?&lt;!\w|-)((?i:begin|break|catch|continue|data|default|define|do|dynamicparam|else|elseif|end|exit|finally|for|foreach(?!-object)|from|if|in|inlinescript|parallel|param|process|return|switch|throw|trap|try|until|var|where(?!-object)|while)|%|\?)(?!\w)</string>
+			<string>(?&lt;!\w|-|\.)((?i:begin|break|catch|continue|data|default|define|do|dynamicparam|else|elseif|end|exit|finally|for|foreach-object|from|if|in|inlinescript|parallel|param|process|return|switch|throw|trap|try|until|var|where-object|while)|%|\?)(?!\w)</string>
```

Comment:
@kborowinski sorry... One more change... I'm an idiot and didn't read the original closely enough. Can you remove `foreach-object` and `where-object` from this entirely. Then it's good to ship :)

Reply:
Just curious because I don't know... What does this change do?

Reply:
it was originally "where(?!-object)" and "foreach(?!-object)" but my brain read it as "(?:-object)" when I asked him to separate where and foreach to their own dict

In fact, @kborowinski when you remove foreach-object and where-object from this dictionary can you add the (?!-object) to the end of both in the one below it. Sorry, my fail.

Reply:
@omniomi Done